debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/snapshot/mode/SchemaRecoveryOnlySnapshotter.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle.snapshot.mode;
+
+import io.debezium.connector.oracle.OracleConnectorConfig;
+
+public class SchemaRecoveryOnlySnapshotter extends SchemaOnlySnapshotter {
```

Comment:
For MySQL there is `SchemaOnlyRecoverySnapshotter`. Is this name intentional or typo?

Reply:
Yes, it was a typo. I have solved with the https://github.com/debezium/debezium/pull/5245. Do you think we can merge this?